LittleHelpBookData/little_help_book.py

```diff
@@ -0,0 +1,100 @@
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, client
+)
+from shared_code.base_scraper import BaseScraper
+
+
+class LHB_Scraper(BaseScraper):
+
+    def scrape_updated_date(self):
+
+        resp = super().scrape_updated_date()
+        soup = BeautifulSoup(resp, 'html.parser')
+        date_string = soup.find('relative-time', class_='no-wrap')
+        date_string = date_string.attrs['datetime']
+        date_string = re.search(
+            r'(19|20)\d\d[- /.](0[1-9]|1[012])[- /.](0[1-9]|[12][0-9]|3[01])',
+            date_string
+        ).group(0)
+        scraped_update_date = datetime.strptime(
+            date_string, '%Y-%m-%d'
+        )
+        return scraped_update_date.date()
+
+
+    def grab_data(self):
+
+        '''Subcategory would be serviceSummary, Servicename would be name, PhoneNumber would be phone,
+        Physical Address should be split into address1, city, state, zip, Hours of operation is schedule,
+         Description can be description, Web Address would be website, Email Address would be contact Email'''
+
+
+        df = pd.read_csv(self.data_url, usecols=self.extract_usecols)
+        df.drop_duplicates(
+            subset=self.drop_duplicates_columns,
+            inplace=True,
+            ignore_index=True
+        )
+        df.rename(columns=self.rename_columns, inplace=True)
+        df['phone'] = df['phone'].str.replace(r'(\d{3})\s*[-.]\s*(\d{3}\s*[-.]\s*\d{4})', r'(\1) \2')
+        df.replace('', np.nan, inplace=True)
+        df.dropna(subset=['name'], inplace=True)
+        df[['address1', 'state', 'zip']] = df['Physical Address'].str.extract(r'(.+)(OR).+(\d{5})', expand=True)
+        df['source'] = self.source
+        df.drop(['Physical Address'], axis=1, inplace=True)
+        return df
+
+
+lhb_scraper = LHB_Scraper(
+    source="LittleHelpBook",
+    data_url='https://github.com/OpenEugene/little-help-book-data/raw/master/data/little-help-book.csv',
+    data_page_url='https://github.com/OpenEugene/little-help-book-data/blob/master/data/little-help-book.csv',
+    data_format="CSV",
+    extract_usecols=[
+        "Subcategory", "Service Name", "Phone Number", 'Hours of operation',
+        'Physical Address', 'Description', 'Web address', 'Email Address'
+    ],
+    drop_duplicates_columns=[
+        "Subcategory", "Service Name", "Phone Number", 'Hours of operation',
+        'Physical Address', 'Description', 'Web address', 'Email Address'
+    ],
+    rename_columns={
+        "Subcategory": "serviceSummary", "Service Name": "name", "Phone Number": 'phone',
+        'Hours of operation': 'schedule', 'Description': 'description', 'Web address': 'website',
+        'Email Address': 'contactEmail'
+    },
+    service_summary="",
+    check_collection="services",
+    dump_collection="tmpPittsburghServices",
+    dupe_collection="tmpPittsburghServicesFoundDuplicates",
```

Comment:
We can drop the word `found` from name of dupe collection because it's already long enough . This can be named as `tmpLHBDuplicates`